jest.config.ts

```diff
@@ -174,7 +174,7 @@ export default {
   ],
 
   // Use this configuration option to add custom reporters to Jest
-  reporters: [['github-actions', { silent: false }], 'summary'],
+  reporters: ['default', ['github-actions', { silent: false }], 'summary'],
```

Comment:
Without this, we weren't seeing the reasons for test failures if something goes wrong (the `default` reporter provides these in console)